arches_querysets/models.py

```diff
@@ -635,6 +672,26 @@ def refresh_from_db(self, using=None, fields=None, from_queryset=None, user=None
             )
         self._refresh_aliased_data(using, fields, from_queryset)
 
+    def backfill_parent_tiles(self):
+        if self.nodegroup.parentnodegroup_id and not self.parenttile_id:
+            self.parenttile = self.parent = TileTree.create_blank_tile(
+                nodegroup=self.nodegroup.parentnodegroup,
+                container=None,
+                permitted_nodes=self._permitted_nodes,
+                caller=self,
+            )
+            return self.parenttile.backfill_parent_tiles()
+        return self
+
+    def _late_fetch_permitted_nodes(self, request):
+        """
+        TODO(next): remove as many of the permitted_nodes variables as possible
+        in favor of request.user.userprofile.{viewable,editable,deletable}_nodegroups.
```

Comment:
Re "next": I do have a branch for #100 that removes the need for this, just leaving this here so that main is stable before I finish that up.